tests/Fixtures/filters/base64_encode.test

```diff
@@ -0,0 +1,10 @@
+--TEST--
+"base64_encode" filter
+--TEMPLATE--
+{{ "foo"|base64_encode|raw }}
+{{ foo|base64_encode|raw }}
+--DATA--
+return ['foo' => new \Twig\Markup('foo', 'UTF-8')]
```

Comment:
why using a Markup object here ?

Reply:
@stof copied test from json_encode can remove it.